README.md

```diff
@@ -1,11 +1,15 @@
-# ios-sample-app
+# XCUITests with A11yUITests for Accessibility Testing on iOS devices
 
 A sample iOS app supporting [XCUI Tests](https://developer.apple.com/library/content/documentation/DeveloperTools/Conceptual/testing_with_xcode/chapters/09-ui_testing.html) on [Browserstack](https://www.browserstack.com/).
 
 <img src="https://cdn-images-1.medium.com/max/1600/1*Z0AH-kvjNsUKlcgjP01rmA.png" height="120" /> ![BrowserStack Logo](https://d98b8t1nnulk5.cloudfront.net/production/images/layout/logo-header.png?1469004780)
 
 ## How to run
```

Comment:
Document all the Pre-requisites like
1. Xcode installation
2. Cocoapods installation
3. Mention Developer Certificate required for iOS App Build.

Reply:
Done